vita3k/gui/src/controllers_dialog.cpp

```diff
@@ -361,6 +361,8 @@ void draw_controllers_dialog(GuiState &gui, EmuEnvState &emuenv) {
         reset_controller_binding(emuenv);
 
     ImGui::End();
+
+    ctrl.drop_inputs = rebinds_is_open; // if the dialog is active, drop inputs from passing to ctrl
```

Comment:
need change this, more good ir probably check in other place  if controll dialog or controller dialog is true set drop to true

Reply:
That's the job of ImGui::Begin(). The p_open parameter that gets passed to ImGui::Begin() is set to TRUE when the window is open and FALSE when the window is closed. placing this parameter outside of ImGui::End() makes certain that when the window is OPEN it will always be TRUE and when it is CLOSED it will always be false. setting drop_binds to the p_open parameter ensures that when the window is open, drop_binds is TRUE and when it is closed, drop binds is FALSE. Same goes for the other place i set drop_binds like this.

Reply:
??? when you close dialog it move value to false
bu no need set this inside function